.github/workflows/run-analysis.yml

```diff
@@ -0,0 +1,31 @@
+name: Run Analysis
+
+# Controls when the action will run.
+on:
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  build_and_run:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v3
+        name: Checkout repo
+
+      - name: Build container and download data
+        uses: ./
+        id: download-data
+        with:
+          args: OPENPBTA_URL=https://s3.amazonaws.com/d3b-openaccess-us-east-1-prd-pbta/data OPENPBTA_RELEASE=testing bash download-data.sh
+
+      - name: Run 01-create-interaction-plots in container
+        uses: ./
+        id: create-interaction-plots
```

Comment:
ID's here need to be unique and not contain numbers. I've made a note of this in the readme as well.